manifests/cf-manifest/operations.d/710-rds-broker.yml

```diff
@@ -3,9 +3,9 @@
   path: /releases/-
   value:
     name: rds-broker
-    version: 0.1.50 
-    url: https://s3-eu-west-1.amazonaws.com/gds-paas-build-releases/rds-broker-0.1.50.tgz
-    sha1: 3de23a9b34eab9c08d337613f594778642c67fa3
+    version: 0.0.1549468886
+    url: https://s3-eu-west-1.amazonaws.com/gds-paas-build-releases/rds-broker-0.0.1549468886.tgz
+    sha1: 18db49a85c2c90837abe0704c2a1839f084ecfe3
```

Comment:
This will need updating once the RDS broker is merged and we've got a build from master.